BUILD

```diff
@@ -6,6 +6,7 @@ sh_binary(
         "MANIFEST.in",
         "setup.py",
         "tensorflow_addons/__init__.py",
+        "tensorflow_addons/version.py",
```

Comment:
I was wondering if we could put those codes in a py_library, say `//tensorflow_addons` ? like tensorflow/probability
 https://github.com/tensorflow/probability/blob/master/BUILD

Reply:
Done.